qiskit_metal/draw/basic.py

```diff
@@ -225,6 +226,14 @@ def _iter_func_geom_(func, objs, *args, overwrite=False, **kwargs):
                                              overwrite=overwrite,
                                              **kwargs)
             return objs
+        elif isinstance(objs, MultiPolygon):
```

Comment:
maybe just add a quick comment in the code why this portion is here (so we don't forget in future)

Reply:
Added comment.

Reply:
@ismirlia , I am not able to see the comment.

Reply:
LGTM , Thanks